appveyor_before_build.ps1

```diff
@@ -0,0 +1,18 @@
+. ./appveyor_helpers.ps1
+
+function PatchCsharpPlugin
+{
+    Add-AppveyorMessage -Message "Copying the C# plugin for patching"
+        
+    $csPluginCleanJar = FindSingleFile "$mavenLocalRepository\org\sonarsource\dotnet\sonar-csharp-plugin" "*SNAPSHOT.jar"
+   
+    $destinationFile = [System.IO.Path]::Combine($env:APPVEYOR_BUILD_FOLDER, "PackagingProjects\CSharpPluginPayload", "csharp_plugin.jar");
+    [System.IO.File]::Copy($csPluginCleanJar, $destinationFile)
+}
+
+
+#Add-AppveyorMessage -Message "Building the latest working C# plugin"
+Build "SonarSource/sonar-csharp" "master"
```

Comment:
I'm going to rename this as "DownloadAndBuildFromGit" as it actually downloads a zip snapshot of master from Git of the specified repo